reaper.go

```diff
@@ -14,6 +14,8 @@ type Config struct {
 	DisablePid1Check bool
 }
 
+var debug bool = false
```

Comment:
Makes more sense to add it to the config structure above. That way we don't need an on/off method - the config variable will drive the debugging messages.

Reply:
That is good idea to expose debug flag in the Config data structure